support-workers/src/main/scala/com/gu/acquisitions/AcquisitionDataRowBuilder.scala

```diff
@@ -123,7 +123,7 @@ object AcquisitionDataRowBuilder {
 
   private def productTypeAndAmount(product: ProductType): (AcquisitionProduct, Option[BigDecimal]) = product match {
     case c: Contribution => (AcquisitionProduct.RecurringContribution, Some(c.amount.toDouble))
-    case s: SupporterPlus => (AcquisitionProduct.SupporterPlus, Some(s.amount.toDouble))
+    case s: SupporterPlus => (AcquisitionProduct.SupporterPlus, None) // we don't send S+ amount because it may be discounted
```

Comment:
don't want to send to fact_acquisition_event as it's not correct, and the analysts are more likely to use zuora data to find out the cost of a subscription.
Not sure if we need to check this change with anyone?  We could alternatively only send it for non discounted subs (it would be hard to know what to send for a discounted sub).  But ideally keep it simple and never send it.

Reply:
It would be worth checking this with Data Design in case they do rely on it for something

Reply:
data design don't use amount at all, they came back and said they check the zuora data in all cases (it sounds like that also includes recurring contribution, but I'll leave that as-is for now, as it's accurate)